src/apof/portal/pipeline.py

```diff
@@ -0,0 +1,8 @@
+def get_avatar(backend, strategy, response, user=None, *args, **kwargs):
+    url = None
+    if backend.name == 'google-oauth2':
+        url = response['image'].get('url')
+    if url:
+        prof = user.profile
+        prof.avatar = url
+        user.save()
```

Comment:
It seems like there is a case that user will be `None` so it will raise attribute error.